Library/Formula/libfixposix.rb

```diff
@@ -0,0 +1,42 @@
+class Libfixposix < Formula
+  env :std
```

Comment:
Otherwise it won't find `autoreconf`
```
==> Checking out tag v0.3.0
==> autoreconf -i -f
Failed to execute: autoreconf
/usr/local/Library/Homebrew/debrew.rb:11:in `raise'
BuildError: Failed executing: autoreconf -i -f
```
Is there another way?